components/usage/cmd/run.go

```diff
@@ -44,12 +45,15 @@ func run() *cobra.Command {
 				log.WithError(err).Fatal("Failed to establish database connection.")
 			}
 
-			err = stripe.Authenticate(apiKeyFile)
-			if err != nil {
-				log.WithError(err).Fatal("Failed to initialize stripe client.")
+			log.Infof("payment integration with Stripe is set to %v", paymentEnabled)
```

Comment:
Logs should always start with a capital letter - it's a sentence intended for humans. Errors on the other hand should always start with a lowercase because they are composed together.